blocks/hooks/custom-class-name.js

```diff
@@ -82,8 +86,8 @@ export function addSaveProps( extraProps, blockType, attributes ) {
 	return extraProps;
 }
 
-export default function customClassName( { addFilter } ) {
-	addFilter( 'registerBlockType', 'core-custom-class-name-attribute', addAttribute );
-	addFilter( 'BlockEdit', 'core-custom-class-name-inspector-control', addInspectorControl );
-	addFilter( 'getSaveContent.extraProps', 'core-custom-class-name-save-props', addSaveProps );
+export default function customClassName() {
+	addFilter( 'blocks.registerBlockType', 'core/custom-class-name/attribute', addAttribute );
```

Comment:
I don't have strong opposition, but noting that there's a few implied conventions being introduced here:

- Scoping filter names by with dot-separated namespaces
   - Any risk of confusion with other namespacing with `/`?
- Multiple levels of namespacing, i.e. not just `core/` but also `core/custom-class-name/`

Reply:
Slash is not allowed for hook names... we need another patch if we want to change that.

Reply:
We can revisit namespaces, too. I’m fine with having only 2 first parts or open to other ideas.